dashboard/test/ui/features/star_labs/share_buttons.feature

```diff
@@ -20,15 +20,15 @@ Feature: Share Buttons
 
   # Making DPad button show up in Game Lab and not in Sprite Lab for mobile
 
-  @no_ie @no_safari @no_firefox @no_chrome
+  @only_mobile
```

Comment:
Also just a shorthand update. This test was having some issues on iPad when I was initially trying to update both iPad and iPhone here (but no issues with just iPhone), but left the shorthand change for better readability.